docs/04-certificate-authority.md

```diff
@@ -135,6 +135,7 @@ cat > ${instance}-csr.json <<EOF
   ]
 }
 EOF
+done
```

Comment:
It already gets closed on what is now line 153 (previously line 152).

Reply:
Ah! You're of course right. Thanks.